p2p/pex/pex_reactor.go

```diff
@@ -387,7 +387,7 @@ func (r *Reactor) ReceiveAddrs(addrs []*p2p.NetAddress, src Peer) error {
 		// If this address came from a seed node, try to connect to it without
 		// waiting (#2093)
 		if srcIsSeed {
-			r.Logger.Info("Will dial address, which came from seed", "addr", netAddr, "seed", srcAddr)
+			r.Logger.Debug("Will dial address, which came from seed", "addr", netAddr, "seed", srcAddr)
```

Comment:
![Screenshot 2021-12-16 at 12 51 27](https://user-images.githubusercontent.com/24299864/146366722-0bd7d1a8-8267-49a0-89dc-735eae701b23.png)
no need to duplicate info

Reply:
maybe omit entirely?

Reply:
removed it